pyproject.toml

```diff
@@ -15,6 +15,7 @@ pytest = "^5.3.4"
 flake8 = "^3.7.9"
 yapf = "^0.29.0"
 jupyter = "^1.0.0"
+requests = "^2.23.0"
```

Comment:
@ManiacMaxo `requests` is not a dev dependency.
Use `poetry add requests`, not `poetry add requests --dev`